experimental/orchestrator/csharp_dotnetcore/01.dispatch-bot/README.md

```diff
@@ -30,7 +30,7 @@ This sample **requires** prerequisites in order to run.
     ```
     - Install bf orchestrator
     ```bash
-    > bf plugins:install @microsoft/bf-orchestrator-cli@beta
+    > bf plugins:install @microsoft/bf-orchestrator-cli@latest
```

Comment:
@feich-ms, we don't need this change.  We'll tag our latest deployment with beta tag.  It's currently being worked on now.  Thanks!

Reply:
sure, reverted